x-pack/plugins/cloud_integrations/cloud_data_migration/public/application/components/app.tsx

```diff
@@ -0,0 +1,165 @@
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React, { useEffect } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiImage,
+  EuiListGroup,
+  EuiListGroupItem,
+  EuiPanel,
+  EuiSpacer,
+  EuiTitle,
+  useEuiTheme,
+} from '@elastic/eui';
+
+import { CoreStart, CoreTheme } from '@kbn/core/public';
+
+import { css } from '@emotion/react';
+import { KibanaThemeProvider, KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
+import { Observable } from 'rxjs';
+import { BreadcrumbService } from '../services/breadcrumbs';
+
+interface CloudDataMigrationAppDeps {
+  http: CoreStart['http'];
+  breadcrumbService: BreadcrumbService;
+  theme$: Observable<CoreTheme>;
+}
+
+export const CloudDataMigrationApp = ({
+  http,
+  breadcrumbService,
+  theme$,
+}: CloudDataMigrationAppDeps) => {
+  const basePath = http.basePath.get() ?? '';
+  const { euiTheme } = useEuiTheme();
+  const listItemProps = {
+    iconType: 'checkInCircleFilled',
+    iconProps: { color: 'success' },
+    wrapText: true,
+  };
+  const paddingCss = css`
+    padding: calc(${euiTheme.size.base} * 6);
+  `;
+
+  useEffect(() => {
+    breadcrumbService.setBreadcrumbs('home');
+  }, [breadcrumbService]);
+
+  return (
+    <KibanaThemeProvider theme$={theme$}>
+      <KibanaContextProvider
+        services={{
+          breadcrumbService,
+        }}
+      >
+        <EuiPanel hasShadow css={paddingCss}>
+          <EuiFlexGroup
+            className="cloudMigration__panel"
+            alignItems="center"
+            gutterSize="xl"
+            justifyContent="spaceBetween"
+          >
+            <EuiFlexItem>
+              <EuiImage
+                alt={i18n.translate('cloudMigration.illustration.alt.text', {
+                  defaultMessage: 'Illustration for cloud data migration',
+                })}
+                className="cloudMigration__illustration"
+                src={`${basePath}/plugins/kibanaReact/assets/` + 'illustration-cloud-migration.svg'}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiTitle size="s">
+                <h3>
+                  <FormattedMessage
+                    id="cloudMigration.migrateToCloudTitle"
+                    defaultMessage="Save time & money by moving your deployment to Elastic Cloud."
+                  />
+                </h3>
+              </EuiTitle>
+
+              <EuiSpacer size="xl" />
+
+              <EuiListGroup maxWidth={700}>
+                <EuiListGroupItem
+                  {...listItemProps}
+                  label={
+                    <FormattedMessage
+                      id="cloudMigration.deployInSeconds.text"
+                      defaultMessage="Deploy in seconds and scale with a click."
```

Comment:
```suggestion
                      defaultMessage="Deploy Elastic quickly and scale up to petabytes of data with a click."
```

This bullet point is light compared to the others but I think maybe leaving it light is the smart thing, since it's a pretty powerful point.